pkg/cmd/container/exec.go

```diff
@@ -181,3 +184,11 @@ func generateExecProcessSpec(ctx context.Context, client *containerd.Client, con
 
 	return pspec, nil
 }
+
+func setInitialTTYSize(pspec *specs.Process) {
+	size, err := console.Current().Size()
+	if err != nil {
+		size.Height, size.Width = 0, 0
```

Comment:
why override with `0, 0` values ? 
may be 
```
	size, err := console.Current().Size()
	if err == nil {
		pspec.ConsoleSize = &specs.Box{Height: uint(size.Height), Width: uint(size.Width)}
       }

Reply:
that's right, I've change codes now